### PR TITLE
Added missing sdkVersion to param

### PR DESCRIPTION
**Pull request by TheOfficialFloW**

### CODE REVIEW

File: include/psp2/common_dialog.h [hunk 1]
```diff
@@ -120,6 +121,7 @@ void sceCommonDialogConfigParamInit(SceCommonDialogConfigParam *param)
 	memset(param, 0x0, sizeof(SceCommonDialogConfigParam));
 	param->language = ~0U;
 	param->enterButtonAssign = ~0U;
+	param->sdkVersion = 0x03150021;
```
Comment (DaveeFTW) [inline include/psp2/common_dialog.h]: What is the meaning of this constant?

**Comment by DaveeFTW** [comment]
This change would imply that existing software using affected structure probably should not work; is this the case?

**Comment by xyzz** [comment]
poke @TheOfficialFloW

**Comment by TheOfficialFloW** [comment]
The constant `0x03150021 `dosen't have any meaning for this param, it is just a value that I have used for in dialogs when I reverse engineered the params in PSM (yes, I didn't have any leaked sdk's back then). (For the message dialog, having this sdk version would allow us to have 1024 chars instead of 512 IIRC).

We can define an other sdk version if suggested (3.60 maybe?).

**Pull request merged**